types/components/Accordion.d.ts

```diff
@@ -4,13 +4,13 @@ import { BsPrefixComponent } from './helpers';
 import AccordionToggle from './AccordionToggle';
 import AccordionCollapse from './AccordionCollapse';
 
-export interface AccordionProps extends React.HTMLProps<Accordion> {
+export interface AccordionProps {
   activeKey?: string;
-  defaultActiveKey?: string
+  defaultActiveKey?: string;
```

Comment:
This should accept a `number` here as well. Better to match with React's Key type: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L78

Reply:
it could but we have cast it to a string regardless b/c we add it to a `data-*` attribute, and don't try attempt to cast back. So ultimately i think its probably simpler to just require a string